worlds/v6/__init__.py

```diff
@@ -0,0 +1,53 @@
+import string
+from .Items import item_table, V6Item
+from .Locations import location_table, V6Location
+from .Options import v6_options
+from ..generic.Rules import set_rule
+from BaseClasses import Region, RegionType, Entrance, Item, MultiWorld
+from ..AutoWorld import World
+
+client_version = 1
+
+
+class V6World(World):
+    """ 
+     VVVVVV is a platform game all about exploring one simple mechanical idea - what if you reversed gravity instead of jumping?
+    """ #Lifted from Store Page
+
+    game: str = "VVVVVV"
+    topology_present = False
+
+    item_name_to_id = item_table
+    location_name_to_id = location_table
+
+    data_version = 1
+    forced_auto_forfeit = False
+
+    options = v6_options
+
+    name_items = {name for name, id in item_table.items()}
+
+    def create_regions(self):
+        reg = Region("Menu", RegionType.Generic, "Dimension VVVVVV", self.player, self.world)
+        for location_name in location_table:
+            location = V6Location(self.player, location_name, location_table[location_name], reg)
+            reg.locations.append(location)
+        self.world.regions.append(reg)
+
+    def set_rules(self):
+        for location_name in location_table:
+            set_rule(self.world.get_location(location_name, self.player), lambda state: True)
+        set_rule(self.world.get_region("Menu", self.player), lambda state: True)
+
+    def create_item(self, name: str) -> Item:
+        item_id = item_table[name]
+        item = V6Item(name, True, item_id, self.player)
+        return item
+
+    def generate_basic(self):      
+        self.world.itempool += list(map(lambda name: self.create_item(name), self.name_items))
```

Comment:
should be able to use self.item_names instead of self.name_items here. In general the form `self.world.itempool += [self.create_item(name) for name in self.item_names]` should be more familiar to most people